.github/workflows/CI.yml

```diff
@@ -50,6 +50,13 @@ jobs:
             ${{ runner.os }}-test-
             ${{ runner.os }}-
       - uses: julia-actions/julia-buildpkg@latest
+      - name: "Compat fix for Julia < v1.5"
+        if: ${{ (matrix.version == '1.3') || (matrix.version == '1.0') }}
+        run: |
+          # Remove Minio from the test project
+          write("Project.toml", read(`grep -v '^Minio =' Project.toml`, String))
+          run(`sed -i -e 's/, "Minio"//' Project.toml`)
+        shell: julia --project=. --startup=no --color=yes {0}
```

Comment:
Why not use `Pkg.rm` for this?

Reply:
because it's in the test deps instead of the regular deps

Reply:
*sigh* Pkg needs to fix this one day